scripts/monorepo-introduction.js

```diff
@@ -69,4 +76,7 @@ async function prepare() {
     }
 }
 
-prepare();
+prepare().catch(e => {
```

Comment:
@zetlen This one causing the build to fail.  Before this PR also there was error but that escaped as `UnhandledPromiseRejectionWarning`  But now this is making the build fail.  

Before - 
$ yarn run -s build
--
196 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
197 |  
198 | ·[0m·[91mlerna ERR! yarn run prepare stderr:
199 | ·[0m·[91mcp: can't create './lib/': Is a directory
200 | error Command failed with exit code 1.
201 | error Command failed with exit code 1.
202 |  
203 | lerna ERR! yarn run prepare exited 1 in 'graphql-cli-validate-magento-pwa-queries'
204 | ·[0m·[91m(node:1204) UnhandledPromiseRejectionWarning: Error: Command failed: lerna --loglevel warn run prepare

On This PR - 
$ yarn run -s build
228 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
229 |  
230 | ·[0m·[91mlerna ERR! yarn run prepare stderr:
231 | ·[0m·[91mcp: can't create './lib/': Is a directory
232 | error Command failed with exit code 1.
233 | error Command failed with exit code 1.
234 |  
235 | lerna ERR! yarn run prepare exited 1 in 'graphql-cli-validate-magento-pwa-queries'
236 | ·[0m·[91mUnexpected error setting up workspace! { Error: Command failed: lerna --loglevel warn run prepare
237 | null
238 | at makeError (/usr/src/app/node_modules/execa/index.js:174:9)
239 | at Promise.all.then.arr (/usr/src/app/node_modules/execa/index.js:278:16)
240 | at process._tickCallback (internal/process/next_tick.js:68:7)
241 | code: 1,
242 | stdout: null,
243 | stderr: null,
244 | failed: true,
245 | signal: null,
246 | cmd: 'lerna --loglevel warn run prepare',
247 | timedOut: false,
248 | killed: false }
249 | ·[0m·[91merror Command failed with exit code 1.
250 | ·[0minfo Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
251 | The command '/bin/sh -c yarn install --frozen-lockfile' returned a non-zero code: 1
252